index.bs

```diff
@@ -139,6 +139,120 @@ The terms [=fulfilled=], [=rejected=], [=resolved=], [=pending=] and
+  </tr>
+  <tr>
+   <td><dfn>receive a datagram</dfn>
+   <td>[[!WEB-TRANSPORT-HTTP3]]
+   [section 4.4](https://datatracker.ietf.org/doc/html/draft-ietf-webtrans-http3/#section-4.4)
+  </tr>
+  <tr>
+   <td><dfn>create an [=stream/outgoing=] stream</dfn>
+   <td>[[!WEB-TRANSPORT-HTTP3]]
+   [section 4.1](https://datatracker.ietf.org/doc/html/draft-ietf-webtrans-http3/#section-4.1)
+  </tr>
+  <tr>
+   <td><dfn>create a [=stream/bidirectional=] stream</dfn>
+   <td>[[!WEB-TRANSPORT-HTTP3]]
+   [section 4.2](https://datatracker.ietf.org/doc/html/draft-ietf-webtrans-http3/#section-4.2)
+  </tr>
+  <tr>
+   <td><dfn>receive an [=stream/incoming=] stream</dfn>
+   <td>[[!WEB-TRANSPORT-HTTP3]]
+   [section 4.1](https://datatracker.ietf.org/doc/html/draft-ietf-webtrans-http3/#section-4.1)
+  </tr>
+  <tr>
+   <td><dfn>receive a [=stream/bidirectional=] stream</dfn>
+   <td>[[!WEB-TRANSPORT-HTTP3]]
+   [section 4.3](https://datatracker.ietf.org/doc/html/draft-ietf-webtrans-http3/#section-4.3)
+  </tr>
+ </tbody>
+</table>
+
+To <dfn for=session>establish</dfn> a [=WebTransport session=], follow [[!WEB-TRANSPORT-HTTP3]]
+[section 3.3](https://datatracker.ietf.org/doc/html/draft-ietf-webtrans-http3/#section-3.3).
+
+To <dfn for=session>terminate</dfn> a [=WebTransport session=] |session| with |code|, run these
+steps:
+
+1. Terminate |session|, as described in [[!WEB-TRANSPORT-HTTP3]]
+   [section 5](https://datatracker.ietf.org/doc/html/draft-ietf-webtrans-http3/#section-5),
+   with using |code| as each RESET_STREAM frame's "Application Protocol Error Code"
+   defined in [[!QUIC]]
+   [section 19.4](https://tools.ietf.org/html/draft-ietf-quic-transport#section-19.4).
+
+<dfn>WebTransport stream</dfn> is a concept for HTTP/3 stream on a [=WebTransport session=].
+
+A [=WebTransport stream=] is one of <dfn for=stream>incoming</dfn>, <dfn for=stream>outgoing</dfn>
+or <dfn for=stream>bidirectional</dfn>.
+
+[=WebTransport stream=] has the following capabilities:
+
+<table class="data" dfn-for="stream">
+ <thead>
+  <tr>
+   <th>capability
+   <th>definition
+   <th>incoming
+   <th>outgoing
+   <th>bidirectional
+  </tr>
+ </thead>
+ <tbody>
+  <tr>
+   <td><dfn>send</dfn> bytes (potentially with FIN)
+   <td>[[!QUIC]]
+   [section 2.2](https://datatracker.ietf.org/doc/html/draft-ietf-quic-transport#section-2.2)
+   <td>No
+   <td>Yes
+   <td>Yes
+  </tr>
+  <tr>
+   <td><dfn>receive</dfn> bytes (potentially with FIN)
+   <td>[[!QUIC]]
+   [section 2.2](https://datatracker.ietf.org/doc/html/draft-ietf-quic-transport#section-2.2)
+   <td>Yes
+   <td>No
+   <td>Yes
+  </tr>
+  <tr>
+   <td><dfn>send STOP_SENDING</dfn>
+   <td>[[!QUIC]]
+   [section 3.5](https://datatracker.ietf.org/doc/html/draft-ietf-quic-transport#section-3.5)
+   <td>Yes
+   <td>No
+   <td>Yes
+  </tr>
+  <tr>
+   <td><dfn>reset</dfn> a [=WebTransport stream=]
+   <td>[[!QUIC]]
+   [section 19.4](https://datatracker.ietf.org/doc/html/draft-ietf-quic-transport#section-19.4)
+   <td>Yes
```

Comment:
RESET_STREAM only works in the outgoing direction, so you can't use it on an incoming stream.  On the other hand, didn't we intend to merge those two into a two-way reset?

Reply:
Fixed. I'm happy to update the description whenever https://datatracker.ietf.org/doc/html/draft-ietf-webtrans-http3/ is updated, but that can be done separately.